index.html

```diff
@@ -5705,6 +5717,62 @@ <h3>Complex Language Markup</h3>
 
     </section>
 
+    <section class="appendix normative">
+      <h2>Verification Algorithm</h2>
+
+      <p>
+This section contains an algorithm that <a>conforming verifier processors</a>
+can run when verifying a <a>verifiable credential</a> or a <a>verifiable
+presentation</a>. Any algorithm can be used that produces output that
+is equivalent to the algorithm defined in this section. This algorithm takes a
+byte stream (<var>inputBytes</var>) and a media type (<var>inputMediaType</var>)
+as inputs, and produces an object (<var>verified</var>) that contains a
+verification status (<var>verified</var>.<var>status</var>),
+a verified document (<var>verified</var>.<var>document</var>),
+a media type (<var>verified</var>.<var>mediaType</var>), zero or more
+optional warnings (<var>verified</var>.<var>warnings</var>), and
+zero or more optional errors (<var>verified</var>.<var>errors</var>).
+      </p>
+
+      <ol class="algorithm">
+        <li>
+Set <var>verified</var> to an empty object.
+        </li>
+        <li>
+Verify that the securing mechanism in use on <var>inputBytes</var> has not been
+tampered with, using <var>inputMediaType</var> as additional input to the
+algorithm. See Section <a href="#securing-verifiable-credentials"></a>
+for details.
+        </li>
+        <li>
+If the verification of the securing mechanism is successful, set
+<var>verified</var>.<var>status</var> to `true`,
+<var>verified</var>.<var>document</var> to the information that was secured,
```

Comment:
"the information that was secured" should be explicitly returned from the Securing method. With the wording that's here now, it's not clear if this is a copy of the input document or a subset that the Securing method actually checked.

Reply:
Your first sentence is the intent, I'll update the PR to make that more clear.

Reply:
Fixed in d45bf08d4e41ca1b7213986f172c0e4055b0251e.